packages/steppers/src/styled/StyledStep.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledStepContent } from './StyledStepContent';
+import { StyledLine } from './StyledLine';
+
+const COMPONENT_ID = 'steppers.step';
+
+export interface IStyledStep {
+  isHorizontal?: boolean;
+}
+
+export const StyledStep = styled.div.attrs<IStyledStep>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledStep>`
+  position: ${props => props.isHorizontal && 'relative'};
+  flex: ${props => props.isHorizontal && '1'};
+
+  &:last-of-type ${StyledLine} {
+    display: ${props => props.theme.rtl && 'none'};
+  }
+
+  &:first-of-type ${StyledLine} {
+    display: ${props => !props.theme.rtl && 'none'};
+  }
```

Comment:
Originally had declared a single selector with string template in the selector, however due to a parse error, the `display` style was separated into using two selectors.

originally:

```js
&:${props => (props.theme.rtl ? 'last-of-type' : 'first-of-type')} ${StyledLine} {
  display: none;
}
```

parseError:

<img width="294" alt="Screen Shot 2020-03-01 at 5 03 51 PM" src="https://user-images.githubusercontent.com/1811365/75705979-4bbda100-5c71-11ea-8674-5273a98de417.png">

Reply:
https://github.com/stylelint/stylelint/issues/4401

Reply:
Tried various `stylelint` disable comments and adding arbitrary CSS selector and styling like the OP in the issue with no luck. Open to suggestions, but I think having a style for both LTR and RTL is okay.